tests/core/test_signal.py

```diff
@@ -281,3 +282,25 @@ async def test_subscription_logs(caplog):
     assert "Making subscription" in caplog.text
     cache.close()
     assert "Closing subscription on source" in caplog.text
+
+
+async def test_signal_lazily_connects(RE):
+    mock_signal_rw = soft_signal_rw(int, "pva://mock_signal", name="mock_signal")
+    await mock_signal_rw.connect(mock=True)
```

Comment:
```suggestion
    await mock_signal_rw.connect(mock=False)
```

since it's a soft_signal_backend

Reply:
ok